Assets/Scripts/IPuzzle.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IPuzzle
+{
+    void AppendSolution(char entry);
```

Comment:
add summaries for these methods. Only add additional summaries in implementations of the interface when they have specific details not covered in the general summaries here.